### ISSUE TITLE
Update of Offline usage description necessary

### ISSUE THREAD

**Issue by chgien** [typingbeaver/snake-label#12]
Hi and thank you very much for this great project!

I tried using the "offline" version just like you describe it in the readme.md -> just open the index.html in the browser.
This does not work like this anymore in actual browsers, the following error message is thrown for example in Chrome for all js files:
`Access to script at 'file:///C:/assets/index.bf17cde9.js' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, isolated-app, chrome-extension, chrome, https, chrome-untrusted.`

In former times, it was no problem to simply load local files in the web browsers, but this had some security issues as noted here:
[https://www.mozilla.org/en-US/security/advisories/mfsa2019-21/#CVE-2019-11730](url)
Therefore the handling in all relevant browsers was changed so trying to load local files now produces CORS errors.
For more info, please refer to this message: [https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS/Errors/CORSRequestNotHttp](url)

A very simple possibility to use as a workaround is to open a simple python HTTP server in the directory where the index.html is located.
This is an easy thing described here: [https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Tools_and_setup/set_up_a_local_testing_server](url)

Long story short: Install python3, then navigate to the directory where index.html is using cd and then call
`python3 -m http.server` or `python -m http.server`. A local http server is now opened at port 8000 (standard port, can be customized).
Then the website can then be opened via webbrowser.

I think, it could help users (especially them with a more or less "basic" IT knowledge) if you would refer to this circumstances in the readme and maybe extend the description with the python http server example, if you like it.

Best regards,
Chris

**Comment by typingbeaver** [comment]
Thanks for bringing this up again! :)

I have now included a singlefile version, so there shouldn't be any CORS problems anymore and without the need of a webserver. (See offline branch)
